src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Utilities/ChecksumExtensions.cs

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+#if !NETCOREAPP
+using System.Collections.Generic;
+#endif
```

Comment:
@davidwengier: Just wanted to note that this is now necessary due to https://github.com/dotnet/razor/pull/9014. Not sure if you expected that for multi-targeted projects.